template/package.json

```diff
@@ -6,30 +6,29 @@
     "dev": "meta dev",
     "build": "meta build",
     "lint": "npm run lint:scripts && npm run lint:styles",
-    "lint:scripts": "eslint web/wp-content/themes/<%= slug %>/src/js --ext=.js,.vue",
-    "lint:styles": "stylelint 'web/wp-content/themes/<%= slug %>/src/**/*.(scss|vue)'"
+    "lint:scripts": "eslint web/wp-content/themes/test2/src/js --ext=.js,.vue",
+    "lint:styles": "stylelint 'web/wp-content/themes/test2/src/**/*.(scss|vue)'"
   },
   "author": "Studio Meta <agence@studiometa.fr> (https://www.studiometa.fr)",
   "license": "MIT",
   "devDependencies": {
-    "@babel/plugin-transform-runtime": "^7.10.5",
-    "@babel/preset-env": "^7.10.2",
+    "@babel/plugin-transform-runtime": "^7.12.1",
+    "@babel/preset-env": "^7.12.1",
     "@fullhuman/postcss-purgecss": "^2.2.0",
```

Comment:
We can delete the Purge CSS package, it is included in the Webpack configuration now.